packages/web/src/common/store/upload/sagas.js

```diff
@@ -76,6 +76,8 @@ const combineMetadata = (trackMetadata, collectionMetadata) => {
 
   if (!metadata.genre) metadata.genre = collectionMetadata.genre
   if (!metadata.mood) metadata.mood = collectionMetadata.mood
+  if (!metadata.release_date)
```

Comment:
just double checking if this is necessary, ie was always a bug or is it passed through in FormTile, or whether just updating release_date to releaseDate fixes it?

Reply:
secret third option:
![image](https://github.com/AudiusProject/audius-protocol/assets/2358254/7df7c21f-65a6-49d9-93de-e93576b47695)

There is no release date field for albums on the old form!

So essentially, yes we do need this line added